
accept_all_with_error_test.exs:9:"test all pass" assert_value "foo" failed

-
+foo

Accept new value? [y,n,?] .

  1) test one fail with error (AcceptAllWithErrorTest)
     accept_all_with_error_test.exs:18
     ** (AssertValue.ExpectedArgumentError) Expected (right) argument should be string, heredoc ("""), or File.read!
     stacktrace:
       accept_all_with_error_test.exs:24: (test)

.
3 tests, 1 failure
